content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md

```diff
@@ -0,0 +1,586 @@
+
+![LoRaWAN® network credentials](assets/CREDENTIALS.png)
+
+As a **gateway** we will be using the [WisGate Edge Lite 2](https://docs.arduino.cc/hardware/wisgate-edge-lite-2), which will provide long-range coverage and access to the network. Learn how to set up yours using this [guide](https://docs.arduino.cc/tutorials/wisgate-edge-lite-2/getting-started).
+
+***If there is coverage of a TTN public gateway in your area, it is not necessary to install yours. You can check your area network [here](https://ttnmapper.org/heatmap/)***
+
+### The Arduino IoT Cloud Dashboard
+
+Taking advantage of the Arduino IoT Cloud, it is possible to seamlessly integrate a simple but powerful dashboard to monitor and visualize the status of the system from remote, resulting in a professional Human-Computer Interaction (HCI) as shown below:
+
+![Arduino Cloud project dashboard ](assets/DASHBOARD2.png)
+
+Within the Arduino IoT Cloud's dashboard, the system variables can be monitored as follow:
+- Each solenoid valve status is shown as **OFF** or **ON**.
+- The activated time of each valve is graphed next to its state.
+- The watermark sensor instant values are shown in gauges alongside a time series record chart.
+- There is a resources section showing the battery voltage in a gauge and the water flow in liters per minute, also, it shows the liters of water used.
+
+![Arduino Cloud project dashboard on a smartphone](assets/DASHBOARD-CELL1.png)
+
+The dashboard is easily accessible from a browser, mobile phone or tablet, allowing a user to receive updates on the irrigation status from anywhere.
+## Full Smart Irrigation System Example
+
+All the necessary files to replicate this application note can be found below:
+
+- The complete code can be downloaded [here](assets/Edge-Control_MKR_Codes.zip).
+
+### The Irrigation System Working
+
+Below you can find some additional images and animations showing how the system works:
+
+![Watermark sensors](assets/WATERMARK.jpg)
+![Battery connection](assets/BATTERY.png)
+![Irrigation in zone three and four](assets/WATER1.gif)
+
+Remember that the system is designed to be scalable and therefore it is possible to control 4 big irrigation zones independently like, for example, different cultivated fields with different crops with different water and soil needs.
+
+## Conclusion
+
+In this application note, you have learned how to build a LoRaWAN® irrigation system to water your crops automatically or manually and monitor the crop's status remotely. Thanks to the soil moisture analysis, you can avoid irrigation when it's not necessary, saving water and avoiding over-irrigation or flooding problems. 
+
+Arduino Edge Control allows you to easily implement this kind of agriculture systems ready for field deployment. Alongside MKR boards, it can get access to the network using the more suitable technology for your application.
```

Comment:
```suggestion
Arduino Edge Control allows you to easily implement this kind of agriculture systems ready for field deployment. Alongside MKR boards, it can get access to the network using the most suitable technology for your application.
```